src/game_inventory.cpp

```diff
@@ -1459,7 +1459,11 @@ class read_inventory_preset: public pickup_inventory_preset
         }
 
         bool is_shown( const item_location &loc ) const override {
-            return loc->is_book() || loc->type->can_use( "learn_spell" );
+            const item_location p_loc = loc.parent_item();
+
+            return ( loc->is_book() || loc->type->can_use( "learn_spell" ) ) &&
+                   ( p_loc.where() == item_location::type::invalid || !p_loc->is_ebook_storage() ||
+                     p_loc->energy_remaining().value() >= 1000000 );
```

Comment:
I think you can do something like this:

```suggestion
                     p_loc->energy_remaining() >= 1000_kJ );
```

Just replace 1000_kJ with the value that allows reading.